### ISSUE TITLE
Make the "materialize all" button more clear when on an asset job page.

### ISSUE THREAD

**Issue by OwenKephart** [dagster-io/dagster#9109]
### What's the use case?

Slack: https://dagster.slack.com/archives/C01U954MEER/p1659137734627489?thread_ts=1659136358.342089&cid=C01U954MEER

Right now, if an asset job has config defined, this config does not get used when hitting the "Materialize all" button on the job's page. This is pretty confusing, as that button really feels like it should do the same thing as executing the job. The basic issue is that "Materialize all" basically just means "Materialize all the assets that happen to be in this job", and in fact the request will end up launching a run of the __ASSET_JOB that we automatically create, rather than the job on that page (even more confusing).

I'm wondering if we should just remove the "Materialize all" button on job pages entirely, or replace it with a "launch job" button, or something else entirely.

### Message from the maintainers

Impacted by this issue? Give it a 👍! We factor engagement into prioritization.

**Comment by sryza** [comment]
I think it should materialize the assets via the job, not via __ASSET_JOB.

Thoughts @bengotow ?

**Comment by bengotow** [comment]
Hey folks! I believe this has already been fixed. To verify, I checked four cases:

- Asset job "Materialize All"
- Asset job "Materialize Selected" (with a subset)
- Asset job where config is required on an asset but not bound to the job
- Asset job where config is bound to `define_asset_job`

In all four cases launching from the Asset Job page started the currently viewed job and not the __ASSET_JOB, so I think we're in good shape here!
